src/NHibernate.Test/NHSpecificTest/GH1228/Fixture.cs

```diff
@@ -5,129 +5,90 @@ namespace NHibernate.Test.NHSpecificTest.GH1228
 	public class Fixture : BugTestCase
```

Comment:
No async counterpart due to missing `[TestFixture]` attribute.

```suggestion
	[TestFixture]
	public class Fixture : BugTestCase
```